dashboard/lib/pd/jot_form/constants.rb

```diff
@@ -10,14 +10,19 @@ module Constants
         TYPE_CHECKBOX = 'checkbox'.freeze,
         TYPE_SCALE = 'scale'.freeze,
         TYPE_MATRIX = 'matrix'.freeze,
+        TYPE_BUTTON = 'button'.freeze,
+        TYPE_DIVIDER = 'divider'.freeze,
+        TYPE_TEXT = 'text'.freeze
```

Comment:
These should be only in the ignored types below, not in the allowed types here.

Reply:
Weird - I thought this is something we fixed. Must have been the bad merge or something I put on top of it